tokio-console/src/warnings.rs

```diff
@@ -78,7 +77,7 @@ impl<T> Linter<T> {
 
     pub(crate) fn format(&self, val: &T) -> String {
         debug_assert!(
-            self.0.check(val),
+            matches!(self.0.check(val), Warning::Ok),
```

Comment:
This should check for `Warning::Warn` instead of `Warning::Ok` (currently `tokio-console` panics every time I try to view a task with a warning on it).
```suggestion
            matches!(self.0.check(val), Warning::Warn),
```

Reply:
whoops, thanks